Examples/MAX78000/ImgCapture/project.mk

```diff
@@ -7,6 +7,12 @@
 
 # **********************************************************
 
+# Add your config here!
+# Uncomment the line below to build for the MAX78000FTHR
+#BOARD=FTHR_RevA
+# Uncomment the line below to build for the MAX78000CAM02
+#BOARD = CAM02_RevA
```

Comment:
Instead of adding the place-holder lines, the preferred suggestion now is to link to the BSP sections of the MSDK User Guide:  https://analogdevicesinc.github.io/msdk//USERGUIDE/#board-support-packages

This covers how to set the BSP for all supported development environments